code/game/gamemodes/blob/blobs/captured_nuke.dm

```diff
@@ -0,0 +1,30 @@
+/obj/structure/blob/captured_nuke //alternative to blob just straight up destroying nukes
+	name = "blob captured nuke"
+	icon = 'icons/mob/blob.dmi'
+	icon_state = "blank_blob"
+	desc = "A Nuclear Warhead tangled in blob tendrils pulsating with a horrific green glow."
+	max_integrity = 100
+	armor = list(MELEE = 0, BULLET = 0, LASER = 0, ENERGY = 0, BOMB = 0, BIO = 0, RAD = 0, FIRE = 65, ACID = 90)
+	point_return = 0
+
+/obj/structure/blob/captured_nuke/Initialize(mapload)
+	. = ..()
+	START_PROCESSING(SSobj, src)
+
+/obj/structure/blob/captured_nuke/adjustcolors(a_color)
```

Comment:
You don't have to remove the overlays each time you adjust the color. When setting the nuke overlay you can set its `appearance_flags` to include `RESET_COLOR`. This makes it keep the original color and thus ignore the parent's color variable. See https://secure.byond.com/docs/ref/index.html#/atom/var/appearance_flags for more info

I also don't see why you set the `icon_state` as `"blank_blob"` and not as `"blob"`. Is it a layering thing? If so, perhaps you can use `underlays` instead of `overlays`.